test/built-ins/RegExp/property-escapes/ASCII.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 Mathias Bynens. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Mathias Bynens
+description: >
+  Unicode property escapes for `ASCII`
+info: |
+  Generated by https://github.com/mathiasbynens/unicode-property-escapes-tests
+  Unicode v9.0.0
+esid: sec-static-semantics-unicodematchproperty-p
+features: [regexp-unicode-property-escapes]
+---*/
+
+const buildString = ({ loneCodePoints, ranges }) => {
+	let result = String.fromCodePoint(...loneCodePoints);
+	for (const [start, end] of ranges) {
+		for (let codePoint = start; codePoint <= end; codePoint++) {
+			result += String.fromCodePoint(codePoint);
+		}
+	}
+	return result;
+};
```

Comment:
if buildString is the same among all tests, we should add a new js file in the `/harness/` folder and get it through the includes tag. i.e.:

```yaml
esid: sec-static-semantics-unicodematchproperty-p
features: [regexp-unicode-property-escapes]
includes: [buildString.js]
---*/
```